examples/util/onnx/OnnxModelImporter.js

```diff
@@ -108,6 +108,46 @@ class OnnxModelImporter {
       }
     }
   }
+  _getLayerData() {
```

Comment:
I understand that you are going to get the converted parameters of NN model instead of the raw data in ONNX model. I would suggest add such a function here.
```js

  getModelParams() {
    return {
      tensorTypes: this._tensorTypes,
      operations: this._operations,
      operands: this._operands
    }
  }
```
so that you can get all the information you want from these three data structures, and it's way easier to use in the test case as they all follow the WebNN API format.

There's a caveat here: since the importer will combine some ONNX nodes into a single node, which means the nodes between two models (ONNX model v.s. WebNN model) are not one-to-one. As a result, you wouldn't be able to get the name of them, something like `squeezenet0_conv12_fwd`. Instead, what you only get is the `opType`, so you may name the test case using the op name with unique ids, like `CONV_0`, `CONV_1`.